### PR TITLE
First run styling pass

### PR DESCRIPTION
**Pull request by bfollington**
image image
image image

https://user-images.githubusercontent.com/5009316/236988506-5e971d90-0f1f-4d9d-9e4c-fd943853b595.mov

https://user-images.githubusercontent.com/5009316/236988541-5f4041ff-f2c5-4e4f-8d43-56fbb843e428.mov

### CODE REVIEW

File: xcode/Subconscious/Shared/Components/AppUpgradeView.swift [hunk 1]
```diff
@@ -86,12 +71,9 @@ struct AppUpgradeView: View {
                     isComplete: state.isComplete,
                     size: spinnerSize
                 ) {
-                    Image("sub_logo")
-                        .resizable()
-                        .frame(width: logoSize, height: logoSize)
+                    StackedGlowingImage(image: Image("sub_logo"), width: logoSize, height: logoSize   )
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Components/AppUpgradeView.swift:74]: Formatting

File: xcode/Subconscious/Shared/Components/FirstRun/FirstRunCreateSphereView.swift [hunk 2]
```diff
@@ -9,25 +9,44 @@ import SwiftUI
...
+    }
+
+    private var shadow: Color {
+        switch colorScheme {
+        case .dark:
+            return .brandBgPurple
+        default:
+            return .brandMarkPurple
+        }
+    }
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Components/FirstRun/FirstRunCreateSphereView.swift:30]: This fragment is repeated a lot, not sure if there's a clever way to extract it?

File: xcode/Subconscious/Shared/AppTheme.swift [hunk 3]
```diff
@@ -122,6 +122,21 @@ extension Color {
     static let brandBgBlack = Color(red: 35/255, green: 31/255, blue: 32/255, opacity: 1) // #231F20
     static let brandBgSlate = Color(red: 57/255, green: 50/255, blue: 84/255, opacity: 1) // #393254
     
+    static let bgGradientLight = LinearGradient(
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/AppTheme.swift:125]: Non-blocking: is this general enough that it ought to be extracted to theme? Or should it be view-specific?

File: xcode/Subconscious/Shared/Components/StackedGlowingImage.swift [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    var body: some View {
+        ZStack {
+            let img = image
+                .resizable()
+                .frame(width: width, height: height)
+            
+            img
+                .blur(radius: 48)
+                .opacity(1)
+                .blendMode(colorScheme == .dark ? .hardLight : .darken)
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/StackedGlowingImage.swift:35]: keanu-reeves-whoa

**Review by gordonbrander — CHANGES_REQUESTED** [review]
One small change requested + questions.

File: xcode/Subconscious/Shared/Assets.xcassets/ns_logo.imageset/Contents.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  "images" : [
+    {
+      "filename" : "noo_logo_light.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
```
[the remark this answers is not in the document]
Comment (bfollington) [reply xcode/Subconscious/Shared/Assets.xcassets/ns_logo.imageset/Contents.json:11]: Great question. I duplicated the `sub_logo` asset and replaced the images, it also says this in the source, I _assume_ it's the mapping of which image to use in light vs dark mode.

File: xcode/Subconscious/Shared/AppTheme.swift (hunk 3 above)
Comment (bfollington) [reply xcode/Subconscious/Shared/AppTheme.swift:125]: I think I'll move it to something more specific like `OnboardingTheme`.

[the remark this answers is not in the document]
Comment (bfollington) [reply xcode/Subconscious/Shared/AppTheme.swift]: Move this to `OnboardingTheme` and took your naming suggestion.

**Review by gordonbrander — APPROVED** [review]
LGTM

**Pull request merged**